views/js/controller/DeliveryMgmt/editDelivery.js

```diff
@@ -19,17 +19,19 @@
  */
 define(['jquery', 'helpers', 'ui/modal'],
 function ($, helpers) {
-	'use strict';
-	
-	return {
+    'use strict';
+
+    return {
         start : function(){
+            var $label = $('#http_2_www_0_tao_0_lu_1_Ontologies_1_TAODelivery_0_rdf_3_Maxexec').prev('label');
+            $label.html($label.text().replace('(', '<br>('));
```

Comment:
I'm not sure this solution is sustainable. Labels are volatiles, can be changed and translated. I understand your issue to place properly the text in the second line.
It's more work but we could add carriage returns characters in the label itself and convert those characters to `<br>`, no ?

Reply:
Or may be just let it as it is, with not `<br>`. In English, without the `<br>`, the label position is correct. And in other languages it can be longer so we may have the label spread over 3 lines.

Reply:
Agreed with second suggestion, as long as there is no whitespace between the bracket and the text this should be fine. 
I left the other chnages in whitespce to satisfy the linter